duckduckgo/iOS
# Add localization to SyncUI package

Task/Issue URL: https://app.asana.com/0/1201493110486074/1204622631026489/f
Tech Design URL:
CC:

**Description**:

Add localization to SyncUI package



**Steps to test this PR**:

Ensure loc update and loc export scripts are running and SyncUI packages are using localized resources.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
